### PR TITLE
More THM docs: materials, actions, scalaraux, vpps, some pps

### PR DESCRIPTION
**Pull request by GiudGiud**
credits to SL express for taking so long

A lot of the objects are kind of trivial so the doco page is quite laminar as well. It's still better than a stub
Please point out the ones you think could use more details or an "example input syntax" to add more meat

### CODE REVIEW

**Comment by GiudGiud** [comment]
Time stepper diff showing up here https://civet.inl.gov/job/1595089/

File: modules/thermal_hydraulics/doc/content/source/postprocessors/MomentumFluxIntegral.md [hunk 1]
```diff
@@ -1,20 +1,14 @@
...
 !syntax description /Postprocessors/MomentumFluxIntegral
 
-## Overview
-
-!! Replace these lines with information regarding the MomentumFluxIntegral object.
+The integral of the momentum flux $I_M$ over boundary $\partial \Omega$ is:
 
-## Example Input File Syntax
+!equation
+I_M = \int_{\partial \Omega} arhouA * velocity + \alpha * p * A d\partial \Omega
```
Comment (joshuahansel) [inline modules/thermal_hydraulics/doc/content/source/postprocessors/MomentumFluxIntegral.md:8]:
```suggestion
I_M = \int_{\partial \Omega} \alpha \rho u^2 A + \alpha p A d\partial \Omega
```

File: modules/thermal_hydraulics/doc/content/source/postprocessors/MomentumFluxIntegral.md [hunk 2]
```diff
@@ -1,20 +1,14 @@
...
 
-!! Describe and include an example of how to use the MomentumFluxIntegral object.
+with $arhouA$ the conserved phase momentum, $velocity$ the boundary 1D velocity,
```
Comment (joshuahansel) [inline modules/thermal_hydraulics/doc/content/source/postprocessors/MomentumFluxIntegral.md:10]:
```suggestion
with $\alpha \rho u A$ the conserved phase momentum, $u$ the boundary 1D velocity,
```

**Review by joshuahansel — CHANGES_REQUESTED** [review]

**Comment by GiudGiud** [comment]
Accepted all, thanks for spotting them!

**Comment by joshuahansel** [comment]
Thanks @GiudGiud/SL Express for doing all of these - I can't imagine it was too exciting...

**Comment by GiudGiud** [comment]
so green so beautiful

**Review by joshuahansel — APPROVED** [review]
So approved

**Pull request merged**